packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -56,18 +56,47 @@ export class CDKDeployer implements BackendDeployer {
       }
     }
 
-    return this.tryInvokeCdk(
+    // first synth with the backend definition
+    const startTime = Date.now();
+    await this.tryInvokeCdk(
+      InvokableCommand.SYNTH,
+      backendId,
+      this.getAppCommand(),
+      cdkCommandArgs.concat('--quiet') // don't print the CFN template to stdout
+    );
+    // CDK prints synth time in seconds rounded to 2 decimal places. Here we duplicate that behavior.
+    const synthTimeSeconds = Math.floor((Date.now() - startTime) / 10) / 100;
+
+    // then run type checks
+    await this.invokeTsc(deployProps);
+
+    // then deploy with the cloud assembly that was generated during synth
+    const deployResult = await this.tryInvokeCdk(
       InvokableCommand.DEPLOY,
       backendId,
+      this.relativeCloudAssemblyLocation,
       cdkCommandArgs
     );
+
+    return {
+      deploymentTimes: {
```

Comment:
before, we were parsing this from stdout of the single deploy command. But now we need to reconstitute it from the two separate commands